modules/effects/arrow-effects/src/main/kotlin/arrow/effects/IOStart.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.effects
+
+import arrow.core.*
+import arrow.effects.internal.*
+import arrow.effects.typeclasses.Fiber
+import kotlin.coroutines.*
+import kotlin.coroutines.Continuation
+
+/**
+ * Create a new [IO] that executes the receiver [IO].
```

Comment:
`Create a new [IO] that eagerly executes the receiver [IO] in the current thread.`

Reply:
I would rather make it `Create a new [IO] that upon execution starts the receiver [IO] within a [Fiber].`